res/resource.rc

```diff
@@ -22,7 +22,7 @@ BEGIN
       VALUE "FileDescription", "A C/C++ implementation of a Sass compiler"
       VALUE "FileVersion", "1.0.0.0"
       VALUE "InternalName", "libsass"
-      VALUE "LegalCopyright", "\251 2017 libsass.org"
+      VALUE "LegalCopyright", "\251 2017 sass-lang.org"
```

Comment:
Not sure about the "LegalCopyright" being held by either domain, but not a blocker to the cleanup

Reply:
Yeah I dunno what these are about, I just did a replace.